matlab_simulation/04-sensor/scanner_bresenham/example.m

```diff
@@ -25,10 +33,11 @@
 % Generate the belief map
```

Comment:
That "creat_a_map.m" function is in the 10-environment folder, I put in there so that everyone could use it as a part of their codes.